src/spec/payid-protocol.md

```diff
@@ -135,17 +135,21 @@ This specification defines the PayID protocol - an application-layer protocol, w
 
       {
        optional string payId,
+       required string version,
        required Address[] addresses,
        optional string memo,
-       optional string identity,
-       optional ProofOfControlSignature proofOfControlSignature
```

Comment:
We have the V. PayID RFC and Self-Sovereign RFCs, so I don't think these need to be in the Base PayID RFC.